### ISSUE TITLE
Crash when app moves to background

### ISSUE THREAD

**Issue by sproctor** [zebra-technologies/Scanner-SDK-for-Android#9]
`connMgr` is null if not detecting classic bluetooth devices. This causes a crash when the app is put into the background.

Decompiled code from AS:
```java
    @OnLifecycleEvent(Event.ON_STOP)
    void onBackground() {
        if (this.connMgr.connectedScanner != null) {
            this.dcssdkEnableBluetoothScannersDiscovery(false);
        }

    }
```

**Comment by sproctor** [comment]
As a workaround, you need to add `sdkHandler.dcssdkSetOperationalMode(DCSSDKDefs.DCSSDK_MODE.DCSSDK_OPMODE_BT_NORMAL)` in the init section regardless of whether you're using normal BT or not.

**Comment by sobels** [comment]
Since my app didn't have BT permissions, I found an alternative workaround. After initializing your `sdkHandler`, call
```kotlin
ProcessLifecycleOwner.get().lifecycle.removeObserver(sdkHandler)
```

**Comment by sproctor** [comment]
Ah, thanks. That's even better.

**Comment by sproctor** [comment]
I got some news from Zebra. This should be fixed in the next release.

**Comment by Riadzebra** [comment]
thank you for your patience 
V2.6.16 has been posted with fix link https://www.zebra.com/us/en/support-downloads/software/developer-tools/scanner-sdk-for-android.html
Release note https://www.zebra.com/content/dam/zebra_new_ia/en-us/software/developer-tools/scanner-sdk-android/release-notes-android-sdk.pdf

**Comment by sproctor** [comment]
Closing, fixed in 2.6.16
